js/lib/urlutil.js

```diff
@@ -441,6 +441,13 @@ const UrlUtil = {
       return parsed.slashes ? [parsed.protocol, parsed.host].join('//') : [parsed.protocol, parsed.host].join('')
     }
     return null
+  },
+
+  stripLocation: (url) => {
+    return url
+      .replace(/\/*$/, '')  // remove trailing /
+      .replace(/#*$/, '') // remove trailing  #
```

Comment:
nit: I think you can combine these into one regex to be faster:
`.replace(/\/?#?$/, '')`

Reply:
Now I saw that it would be great to first remove `#` and then `/`, so that would link like `https://tweetdeck.twitter.com/#` become `https://tweetdeck.twitter.com`.

So old code would be changed to:
```
return url
      .replace(/#*$/, '') // remove trailing  #
      .replace(/\/*$/, '')  // remove trailing /
```

@bbondy Can you please update regex to reflect that change. I played around, but could find correct regex?